src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -38,6 +38,10 @@
  */
 class TranslationDebugCommand extends Command
 {
+    const EXIT_CODE_GENERAL_ERROR = 1;
+    const EXIT_CODE_MISSING = 2;
+    const EXIT_CODE_UNUSED = 4;
+    const EXIT_CODE_FALLBACK = 8;
```

Comment:
Console exit codes are kind of standardized (see https://www.tldp.org/LDP/abs/html/exitcodes.html). I propose to use 64 and above here.

Reply:
Good catch! I have changed these to >=64.